src/stories/Composer.stories.tsx

```diff
@@ -1,4 +1,5 @@
 import type {Meta} from '@storybook/react';
+// eslint-disable-next-line no-restricted-imports
 import {ExpensiMark} from 'expensify-common';
```

Comment:
Because it's for storybook, and I believe we don't need (or even have) Onyx context for it. Do you think it would make sense to use Parser here as well?